<?xml version="1.0" ?>

<launch>
  <param name="/use_sim_time" value="true"/>

  <!-- Robot 1 Ground Truth Publisher -->
  <group ns="robot1">
    <arg name="tf_prefix" value="robot1_tf" />

    <param name="frame_id" value="map"/>
    <param name="robot_name" value="Robot1"/>
    <param name="namespace_" value="robot1"/>
    <node pkg="experiment_launcher" type="gazebo_ground_truth" name="gazebo_ground_truth"/>
  </group>


  <group ns="robot2">
    <arg name="tf_prefix" value="robot2_tf" />

    <param name="frame_id" value="map"/>
    <param name="robot_name" value="Robot2"/>
    <param name="namespace_" value="robot2"/>
    <node pkg="experiment_launcher" type="gazebo_ground_truth" name="gazebo_ground_truth"/>
  </group>

  <group ns="robot3">
    <arg name="tf_prefix" value="robot3_tf" />

    <param name="frame_id" value="map"/>
    <param name="robot_name" value="Robot3"/>
    <param name="namespace_" value="robot3"/>
    <node pkg="experiment_launcher" type="gazebo_ground_truth" name="gazebo_ground_truth"/>
  </group>

</launch>
